### PR TITLE
add new test set to saferdialogues task

### PR DESCRIPTION
**Pull request by meganung**
**Patch description**
add the newly collected test set that is parallel with the BAD test set.

### CODE REVIEW

File: parlai/tasks/saferdialogues/README.md [hunk 1]
```diff
@@ -16,4 +16,6 @@ Note: The dataset is flattened, so there is one example per episode.
 
 If the `--recovery` flag is set to `false` (`true` by default) then the recovery response is omitted and the labels contains the signaling message and the text contains the context lines before that.
 
+Use the `--bad` flag and set to `true` (`false` by default) to use a different test set that is paralell to the BAD dataset's test split.
```
Comment (jxmsML) [inline parlai/tasks/saferdialogues/README.md:19]: is the new addition just for test split, or for both valid and test?

Comment (meganung) [reply parlai/tasks/saferdialogues/README.md:19]: just for test split

Comment (stephenroller) [inline parlai/tasks/saferdialogues/README.md:19]: Spelling

File: parlai/tasks/saferdialogues/agents.py [hunk 2]
```diff
@@ -35,6 +37,12 @@ def add_cmdline_args(
             default=True,
             help="Whether or not to include the recovery utterance",
         )
+        agent.add_argument(
+            '--bad',
```
Comment (stephenroller) [inline parlai/tasks/saferdialogues/agents.py:41]: I kinda suggest we make a new teacher instead of hiding this behind a flag

**Pull request closed without merging**